website/templates/navbar.html

```diff
@@ -1,2 +1,8 @@
-<!-- your html code here -->
-<p>Hello, I'm the navbar.html</p>
\ No newline at end of file
+<nav>
+    <ul>
+        <li>Start Bootstrap</li>
```

Comment:
Well... this actually has a different size and font weight in the design, can you adjust it, please?